src/FlowtypeRunner.js

```diff
@@ -0,0 +1,48 @@
+const { exec } = require('child_process');
+const path = require('path');
+const { pass, fail } = require('create-jest-runner');
+
+class FlowtypeRunner {
+  constructor(globalConfig) {
+    this.globalConfig = globalConfig;
+  }
+
+  runTests(tests, watcher, onStart, onResult, onFailure, options) {
+    this.globalConfig = {};
+    const start = +new Date();
+    return new Promise((resolve) => {
+      exec('flow', { stdio: 'ignore', cwd: process.cwd() }, (err, stdout) => {
+        const errors = stdout.split('Error');
+        const errorsPerFile = errors.reduce((previous, current) => {
+          const firstErrorLine = current.split('\n')[0];
+          let fileName = firstErrorLine.split(':')[1];
+          if (fileName) {
+            fileName = path.join(process.cwd(), fileName.trim());
+            const errorMessage = current.substring(current.indexOf('\n') + 1);
+            if (!previous[fileName]) {
+              previous[fileName] = [];
+            }
+            previous[fileName].push(errorMessage);
+          }
+          return previous;
+        }, {});
+        tests.map((t) => {
+          if (!errorsPerFile[t.path]) {
```

Comment:
Testing for a negative can be misleading in some situations. It might be better to switch this if/else around if possible.

`if (errorsPerFile[t.path])...`

Reply:
That's right. Thank you